staging/src/k8s.io/apimachinery/pkg/api/validation/path/name_test.go

```diff
@@ -57,7 +57,7 @@ func TestValidatePathSegmentName(t *testing.T) {
 		},
 		// Make sure non-ascii characters are tolerated
```

Comment:
Reading this comment, it seems to be intentional to use non-ascii characters.

Reply:
Got it ，i will close  this pr